bot/cogs/help.py

```diff
@@ -70,499 +72,306 @@ class HelpSession:
     the regular description (class docstring) of the first cog found in the category.
     """
 
-    def __init__(
-        self,
-        ctx: Context,
-        *command,
-        cleanup: bool = False,
-        only_can_run: bool = True,
-        show_hidden: bool = False,
-        max_lines: int = 15
-    ):
-        """Creates an instance of the HelpSession class."""
-        self._ctx = ctx
-        self._bot = ctx.bot
-        self.title = "Command Help"
-
-        # set the query details for the session
-        if command:
-            query_str = ' '.join(command)
-            self.query = self._get_query(query_str)
-            self.description = self.query.description or self.query.help
-        else:
-            self.query = ctx.bot
-            self.description = self.query.description
-        self.author = ctx.author
-        self.destination = ctx.channel
-
-        # set the config for the session
-        self._cleanup = cleanup
-        self._only_can_run = only_can_run
-        self._show_hidden = show_hidden
-        self._max_lines = max_lines
-
-        # init session states
-        self._pages = None
-        self._current_page = 0
-        self.message = None
-        self._timeout_task = None
-        self.reset_timeout()
-
-    def _get_query(self, query: str) -> Union[Command, Cog]:
+    def __init__(self):
+        super().__init__(command_attrs={"help": "Shows help for bot commands"})
+
+    @redirect_output(destination_channel=Channels.bot, bypass_roles=STAFF_ROLES)
```

Comment:
Some constants were renamed recently.
```suggestion
    @redirect_output(destination_channel=Channels.bot_commands, bypass_roles=STAFF_ROLES)
```